os/dutils.sh

```diff
@@ -154,6 +214,8 @@ function MENU()
 			echo "  \"Select Option\""
 			echo ======================================================
 			echo "  \"1. Run Toolchain Command\""
+			echo "  \"2. Run TRAP Debug tool\""
```

Comment:
Let's change the numbers.
"1. Run TRAP ...."
"2. Run Toolchain ...."
When I imagine other commands, usually 1st option is recommended.
When user wants to use individual single command, then it's next option.

Reply:
Done